fragments/tap-workload/accelerator.yaml

```diff
@@ -1,25 +1,27 @@
 engine:
-  condition: "#bsGitRepository != null"
```

Comment:
I think it would be better to move the `condition` to the merge child, ie

```yaml
engine:
  merge:
    - includes: [ "**/workload.yaml" ]
      condition: "#bsGitRepository != null"
      onConflict: NWayDiff
      merge:
        - type: ReplaceText # No-Op. In combination with NWayDiff, allows keeping the workload.yaml
          # file untouched if the $.spec.source.git.url is NOT present (OpenRewrite emits nothing otherwise)
...
```

Reply:
We have introduced a `chain` instead of a `merge` at the beginning of this fragment, so not clear why your example is provding a `merge`. 

Besides that, I think having a condition at the beginning of a branch is very clear for the next developer who needs to change or maintain it.

Reply:
can you please be more specific and explain why this would be better?
In my opinion having a condition as a first line helps reading this accelerator definition. Reader knows straight away under what condition following block is going to be executed and they can decide whether this branch is of interest for them or not.